handsontable/handsontable
# UndoRedo should track changes only if a new value is different than the previous one

### Context
This PR changes the way how UndoRedo plugin track changes in cells. Now it will verify if the change stack contains change with different values in `oldVal` and `newVal` arguments. If there is at least one difference, then UndoStack will save this change action.

### How has this been tested?
1. Select `A1`
2. Insert a new value into `A1`
3. Select `A2`
4. Open a cell editor
5. Close the cell editor with <kbd>ENTER</kbd> (don't change its value)
6. Use <kbd>CTRL / CMD</kbd> + <kbd>Z</kbd> shortcut to revert changes

Result: `A1`'s value should be revert to `A1`.

### Types of changes

### Related issue(s):
1. #4072
